ota-plus-web/docs/modules/ROOT/pages/get-access.adoc

```diff
@@ -15,7 +15,7 @@ NOTE: HERE needs to verify your date of birth to ensure that you are of the lega
 6. In *Security check*, enter the security code that you see in the picture.
 +
 [.lightbackground]
-image::img::log_in_dialog_box.png[image, align="left"]
+image::img::log_in_dialog_box.png[image,500,align="left"]
```

Comment:
Probably ok, but it's better to use percentages than pixels.

Reply:
Changed